src/__test__/NavBar.test.jsx

```diff
@@ -0,0 +1,55 @@
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import NavBar from '../components/navbar/NavBar';
```

Comment:
- [ ] Hi @haftamudesta, I suggest you double-check your code before submitting. The previous reviewers points out this issue and it seems you didn't take it into consideration. You seem to import a module that doesn't exist. Kindly replace the line(**import NavBar from '../components/navbar/NavBar';**) with the line(**import classes from './Navbar.module.css';**)
- The difference here is that you are importing `Nav**B**ar` instead of `N**a**bar`
- The short video below will help you to import files that already exist in your local directory. After clicking **'./'** VS code(If it is your IDE) would help you to propose the folders and files in that directory.

[Screencast from 06-02-2023 09:10:59 AM.webm](https://github.com/haftamudesta/React-Capstone-Project/assets/63087491/a66ec4c3-0a2a-4d03-9095-16eb1874bf28)